pymc_experimental/tests/test_model_builder.py

```diff
@@ -129,6 +149,12 @@ def test_predict():
     assert isinstance(pred["y_model"][0], float)
 
 
+def test_predict_posterior():
+    data, model_config, sampler_config = test_ModelBuilder.create_sample_input()
+    model = test_ModelBuilder(model_config, sampler_config, data)
+    model.fit()
+
```

Comment:
should have an assert here, even if it's just a simple one.

Reply:
I've added them, and while adding I've noticed that I'd be introducing 4 or 5 repetitions of the same few lines so threw in a helper function, take a look at it in free moment and let me know if you like the end result.